### ISSUE TITLE
Add Classification as a field on GenericAlert?

### ISSUE THREAD

**Issue by Fingel** [TOMToolkit/tom_base#479]
Hi,

Many if not most of the alert brokers now (Lasair, Alerce, Antares, etc) provide some kind of ML classification along with their alerts. This is super helpful information to have when deciding which alerts to import into a TOM. I think it would be helpful to add "classification: str" as a field on the GenericAlert class so that it can be set by the individual broker classes and displayed in the list when you run a broker query.

Thoughts?

**Comment by dmcollom** [comment]
Yes, I completely agree, although there's just a touch of ambiguity in that there are different levels of classification. I doubt that would be a problem, though. We can add this to the project board and see if there's time in the next sprint.
